jarbas/chamber_of_deputies/views.py

```diff
@@ -77,25 +44,37 @@ def _build_filters(self):
         if in_latest is not None:
             self.queryset = self.queryset.in_latest_dataset(in_latest)
 
+        # filter search_vector
+        vector_param = self._vector_param()
```

Comment:
This `self._vector_param()` could be replaced by `self.request.query_params.get('search')` (and we wouldn't need the auxiliar method). It's up to you : )